### PR TITLE
Return stack trace printing for FatalErrors

### PR DESCRIPTION
**Pull request by brendongo**
PR checklist:
- [ ] documentation is up to date
- [ ] changelog is up to date

### CODE REVIEW

File: semgrep/semgrep/core_output.py [hunk 1]
```diff
@@ -122,14 +123,18 @@ def parse(cls, raw_json: JsonObject) -> "CoreError":
             level_str = "WARN"
         level = Level[level_str.upper()]
 
+        details = raw_json.get("details")
```
Comment (emjin) [inline semgrep/semgrep/core_output.py:126]: Wait what does this do if details isn't there?

Comment (brendongo) [reply semgrep/semgrep/core_output.py:126]: Returns None

Comment (brendongo) [reply semgrep/semgrep/core_output.py:126]: So https://github.com/returntocorp/semgrep/pull/3876#diff-ac75fcfbab642d994ff6593c78b023777d5cd0bb672397b73bbe950f96e26bc6R145 will be used

**Review by emjin — APPROVED** [review]
Does this use Iago's nice printing?

**Comment by brendongo** [comment]
Yeah I followed the format in https://github.com/returntocorp/semgrep/pull/3809 but kept it just all red cause white text probably wont show up nicely on white terminals.

**Comment by emjin** [comment]
> Yeah I followed the format in #3809 but kept it just all red cause white text probably wont show up nicely on white terminals.

I remember the text being a light gray, which I thought worked nicely (doesn't overwhelm the person running Semgrep, but they can see if it they need)

**Comment by IagoAbal** [comment]
> Yeah I followed the format in #3809 but kept it just all red cause white text probably wont show up nicely on white terminals.

See https://github.com/returntocorp/semgrep/pull/3809#issuecomment-912363781. I couldn't find gray color defined in that Python library but it seems to do the right thing depending on the color theme of the terminal.

**Review by IagoAbal — APPROVED** [review]

**Comment by brendongo** [comment]
I see. I'll change it back to white

**Pull request merged**